### PR TITLE
fix: add project structure to readme

### PR DESCRIPTION
**Pull request by ankitjena**
#17 Add the following template structure
```
|-----generated
       |-----schema.graphql
       |-----resolvers.ts
|-----model
       |-----<your-model-name>.graphql
|-----src
       |-----config
             |-----config.ts
       |-----db.ts
       |-----index.ts
       |-----mapping.ts
|-----package.json
|-----Dockerfile
|-----docker-compose.yml
|-----package-lock.json
|-----tslint.json
|-----README.md
```

### CODE REVIEW

File: templates/apollo-starter-ts/README.md [hunk 1]
```diff
@@ -7,3 +7,25 @@ Starter template using graphback and apollo-server
 docker-compose up -d
 npm start
 ```
+
+### Project structure
+
```
Comment (wtrocki) [inline templates/apollo-starter-ts/README.md]: Without explanation this makes no sense. I will put typical folder structure. we need the same in docs. We are missing cli in docs at the moment

File: templates/apollo-starter-ts/README.md [hunk 2]
```diff
@@ -7,3 +7,25 @@ Starter template using graphback and apollo-server
...
+```
+|-----generated
+       |-----schema.graphql
+       |-----resolvers.ts
+|-----model
+       |-----<your-model-name>.graphql
+|-----src
```
Comment (wtrocki) [inline templates/apollo-starter-ts/README.md]: i will just explain root. rest will change so often

File: templates/apollo-starter-ts/README.md (hunk 1 above)
Comment (ankitjena) [reply templates/apollo-starter-ts/README.md]: Okay, so close this?

File: templates/apollo-starter-ts/README.md (hunk 2 above)
Comment (ankitjena) [reply templates/apollo-starter-ts/README.md]: Just the directory structure, not the content inside

File: templates/apollo-starter-ts/README.md (hunk 1 above)
Comment (wtrocki) [reply templates/apollo-starter-ts/README.md]: Absolutely not. I would never say no for more documentation especialy that we need to document this, however we need to put here more than just result from `three`command. 

I think we should agree on our strategy to document entire project.

We can be:

A) Command line centric
B) Concept centric with workflow. 

I think that we should have some kind of getting started documentation 
where we will go with Init. Pick one of the templates and execute points that were printed in init command. Project structure is cool but I would love to have just more text explaining the steps and how to threat this project.

**Comment by wtrocki** [comment]
Status of this?

**Pull request merged**